src/pks/transport/sediment_transport/sediment_transport_pk.cc

```diff
@@ -119,6 +119,7 @@ void SedimentTransport_PK::Setup(const Teuchos::Ptr<State>& S)
 
   water_tolerance_ = tp_list_->get<double>("water tolerance", 1e-1);
   max_tcc_ = tp_list_->get<double>("maximal concentration", 0.9);
+  sediment_density_ = tp_list_->get<double>("sediment density [kg/m^3]");
```

Comment:
Can you make this "sediment density [kg m^-3]".  Some of our xml parsing chokes on "/" in names, so all ATS unit strings at this point have been changed to negative exponents rather than divide (which is a bit confusing from an order of operations standpoint anyway, as it requires parentheses in some cases).